airsonic-main/src/test/java/org/airsonic/player/util/MigrationConstantsRule.java

```diff
@@ -0,0 +1,12 @@
+package org.airsonic.player.util;
+
+import org.airsonic.player.Application;
+import org.junit.rules.ExternalResource;
+
+public class MigrationConstantsRule extends ExternalResource {
```

Comment:
New rule to set migration constants for the environment in tests (done in Application.java for regular application start up)